### DISCUSSION TITLE
Publications by categories

### DISCUSSION THREAD

**Discussion by dxvidmr** [alshedivat/al-folio#485] [Q&A] [answered]
Hello! Could someone help me sort my publications by categories? I think I have to add the `categories` in the `publications.md` file, but I'm a bit lost.

Any help will be appreciated... thanks!

**Comment by pdeperth** [comment, marked as answer]
Hello,
you can change the `group_by` parameter in the Jekyll Scholar section of the `_config.yml` file to:
`group_by: type, year`
Best.

**Reply by dxvidmr** [reply]
Thanks!
